### ISSUE TITLE
Unix sensitivity, confuses different client timestamps as separate timestamps

### ISSUE THREAD

**Issue by hzadeh17** [ui-libraries/flint#23]
I would like to use unix timestamps as a method of at least quantifying the number of unique emails or, conversely, the number of duplicates. In a list of all the unique unix timestamps in the corpus should give you an estimate of the total number of unique emails (as I have done--the estimate right now is 9027).

The problem is that the unix converter is specific to seconds, so two identical emails from different clients might show the timestamps 5:53 PM and 5:53:20 PM, which will be read as two unique emails in the unix-duplicate-finder method.

**Comment by akrinaldi** [comment]
Would looking at whether the senders/receivers were the same for the two help at all or would that create more confusion?

**Comment by hzadeh17** [comment]
So what I've done so far is created a dict.fromkeys list (no duplicates) from a list of unix ts rounded to the tens place. This means that any ts within 100 seconds of eachother one will be detected as the same, or a duplicate.

But it would be a good idea to double check this method and see if these have the same senders/receivers--I will try to work on this!

**Comment by akrinaldi** [comment]
Do you have the output for that uploaded yet? If so, where? I can make it into a table and we can check senders/receivers using the searchable table.

**Comment by hzadeh17** [comment]
Okay, so this is what I have so far: [raw](https://raw.githubusercontent.com/ui-libraries/flint/master/dupl_compare.txt),[ on GitHub](https://github.com/ui-libraries/flint/blob/master/dupl_compare.txt) (this is for all deqs)

This is a big list, with each item being: (deqfilename.txt, list of rounded unix ts on page, list of senders on page). So, in theory, if the page is an exact duplicate then item[1] and item[2] should be the same. However, it is more complicated if we are just thinking about duplicate emails (in a different thread context). 

I am still working on automating a process of going in and finding out whether duplicates of the same ts are matched to the same sender, but at first glance it seems that a few of them are not...

**Comment by hzadeh17** [comment]
Okay, [here'](https://github.com/ui-libraries/flint/blob/master/dupl_compare2(2).txt)s the progress. This list is dictionary, where the key is the rounded ts and the value are all the sender names associated with it. (Ignore the 0 key at the beginning, those are the sender names with no associated ts).

Maybe not much to do with th

**Comment by akrinaldi** [comment]
I used that data to make a table we can organize to see if we're getting repeats of senders/receivers. It's [here](https://github.com/ui-libraries/flint/blob/master/visualizations/tables/tdupes.html).

**Comment by hzadeh17** [comment]
@akrinaldi great! Sorry if this is obvious, but how do I see the table from the code?

Also, I finally ran everything through to confirm if duplicates had the same sender names. I explained/documented the process [here](https://docs.google.com/document/d/13-q_Mc_3YXDLBTRx6QnVo6iTz41IuikLLQ9RkatLCnM/edit). 

It's kind-of confusing, but basically this is just a new method that asks both _a_. are two rounded timestamps the same? and if so _b_. are they associated with the same sender? If the answers to _a_ and _b_ are yes, they register as a duplicate. With this method, the new estimate for unique emails is 8801 (up from 7629, which only asked question _a_).
